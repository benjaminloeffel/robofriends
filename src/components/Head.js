import React from "react";
import { Helmet } from "react-helmet";

const Head = () => {
  return (
    <Helmet>
      <meta charSet="utf-8" />
      <meta name="description" content="Search through your Robot Friends" />
      <meta
        name="viewport"
        conent="width=device-width, initial-scale=1, shrink-to-fit=no"
      ></meta>

      {/* PWA */}
      <link
        rel="apple-touch-icon"
        sizes="180x180"
        href="/apple-touch-icon.png"
      ></link>
      <link
        rel="icon"
        type="image/png"
        sizes="32x32"
        href="/favicon-32x32.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="16x16"
        href="/favicon-16x16.png"
      />
      <link rel="manifest" href="/manifest.json" />
      <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#ec5b31" />
      <meta name="msapplication-TileColor" content="#ec5b31" />
      <meta name="theme-color" content="#ec5b31"></meta>

      <title>RoboFriends</title>
    </Helmet>
  );
};

export default Head;
