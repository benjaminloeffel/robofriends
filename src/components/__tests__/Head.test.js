import React from "react";
import "../../setupTests";
import { shallow } from "enzyme";
import Head from "../Head";

describe("Head", () => {
  const wrapper = shallow(<Head />);

  it("renders the Head component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
