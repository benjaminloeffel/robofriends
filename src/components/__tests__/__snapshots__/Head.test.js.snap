// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders the Head component 1`] = `ShallowWrapper {}`;
